discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -580,6 +582,17 @@ def wait_for_hosts_to_install(
         )
 
     def wait_for_operators_to_finish(self, timeout=consts.CLUSTER_INSTALLATION_TIMEOUT, fall_on_error_status=True):
+        operators = self.get_operators()
+        utils.wait_till_all_operators_are_in_status(
+            client=self.api_client,
+            cluster_id=self.id,
+            operators_count=len(utils.filter_operators_by_type(operators, OperatorType.BUILTIN)),
+            operator_type=OperatorType.BUILTIN,
+            statuses=[consts.OperatorStatus.AVAILABLE],
+            timeout=timeout,
+            fall_on_error_status=fall_on_error_status,
```

Comment:
we shouldn't have fall_on_error_status=True for built-in

Reply:
I switched the statuses.
In case someone wants to use the API and fail on error (when only available is acceptable for builtins) then it should work. (for example, to test failures)
Regarding OLMs we accept anyway both of them.